### DISCUSSION TITLE
Boost button for dashboard

### DISCUSSION THREAD

**Discussion by stephencrowther** [CJNE/ha-myenergi#603] [Q&A]
New to HA. How do I create a dashboard button/interface where I can invoke a manual boost choosing the time period to boost for?

Thanks

**Comment by LeiChat** [comment]
Here's the "tap action" for a dashboard button that calls the boost service for my eddi, passing it a length of time in minutes to boost for. Haven't created similar for my zappi but I presume it would be along similar lines. 

It's the YAML code but you should hopefully be able to do it through the HA edit dashboard user interface. 
![Screenshot_2024-11-16-20-12-44-44_c3a231c25ed346e59462e84656a70e50](https://github.com/user-attachments/assets/e0302517-1187-47a5-a408-08308e38eed0)

**Reply by stephencrowther** [reply]
Thanks.

That gives a simple fixed boost, but I was wondering if there was a way of something more interactive that gives you a way of entering the boost length as you activate it - maybe with a slider input or something - no clue how to do that.

**Reply by LeiChat** [reply]
You could create a Number 'Helper', with min, max and step range. Add that to the dashboard as a slider/drop down next to a button that triggers the action using the value of the helper.

**Reply by LeiChat** [reply]
![Screenshot_2024-11-16-23-04-23-79_c3a231c25ed346e59462e84656a70e50](https://github.com/user-attachments/assets/b0af32b5-7daf-42b7-8454-33d30025857c)
